### DISCUSSION TITLE
install GMTSAR issues

### DISCUSSION THREAD

**Discussion by Congzhangg** [gmtsar/gmtsar#557] [Q&A] [answered]
When I installed GNTSAR to the autoupdate command, why did I get an error like the one below, causing subsequent installation of the software to fail
![1668591761244](https://user-images.githubusercontent.com/112800803/202146193-5d0ef5b6-7dbc-40cb-ba81-abaab467ffc8.png)

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
I don't think the following installation will fail because of a obsolete macro. Please continue your installation with `./configure`, `make` and `make install`.

**Reply by Congzhangg** [reply]
Okay, I tried several times a few hours ago, even trying to switch sources, but it didn't work. We just went ahead and installed it successfully, Thank you for your advice.

**Reply by Xiaohua-Eric-Xu** [reply]
The macro from autoconf is outdated, and that's where the warning come from. Still it is just a warning.

**Reply by wajidraki** [reply]
I also had this problem twice. I didn't get a config error. when I run make and make install. I get an error "make [2] : [Makefile:10 : ext_orb_lt1] error 1"

**Reply by Xiaohua-Eric-Xu** [reply]
You may need to attach more details for us to know what happened. A log file would be really helpful.
